make.xsh

```diff
@@ -170,10 +173,25 @@ def start_database_stack():
     $POSTGRES_HOST = get_localhost()
     # Deletes any existing pgdata folder and reinitiates it.
     rm -rf infrastructure/containers/postgres/data/pgdata/
-    docker network prune --force
     docker swarm init || echo "docker swarm is already initialized :D"
     docker stack deploy --compose-file docker-compose.yml rootski-database
 
+@makefile.target(tag="run services locally")
```

Comment:
can we call this `start-database-stack-prod`? I feel like there's no reason we wouldn't exclude contributors with database contributors from running this on their local machines. I don't think the logic is specific to this being on lightsail at all.